packages/commonwealth/server/models/subscription/subscription.ts

```diff
@@ -0,0 +1,94 @@
+import Sequelize, { DataTypes} from 'sequelize';
```

Comment:
Why was this moved into a folder?

Reply:
This needed to be moved out of the Subscriptions class, because it was causing tests relying on the db (which relied on the Subscription model) to take quite a long time to load. If we split this method from Subscription, only tests that rely on this emitNotification feature will take a long time to load.

Reply:
Now that emitNotification is in utils, we can move this back into the models folder?